addons/product/models/product_product.py

```diff
@@ -51,6 +53,14 @@ class ProductProduct(models.Model):
     product_template_attribute_value_ids = fields.Many2many('product.template.attribute.value', relation='product_variant_combination', string="Attribute Values", ondelete='restrict')
     product_template_variant_value_ids = fields.Many2many('product.template.attribute.value', relation='product_variant_combination',
                                                           domain=[('attribute_line_id.value_count', '>', 1)], string="Variant Values", ondelete='restrict')
+
+    import_attribute_values = fields.Char(
+        store=False,
+        copy=False,
+        compute='_export_attribute_values',
+        inverse='_import_attribute_values',
+        string='Product Values')
```

Comment:
Only for the sake of readability
```suggestion
    import_attribute_values = fields.Char(
        string='Product Values',
        compute='_export_attribute_values',
        inverse='_import_attribute_values',
        store=False, copy=False)
```